test/unit/org/apache/cassandra/db/CellTest.java

```diff
@@ -416,20 +416,32 @@ private Cell<?> regular(TableMetadata cfm, String columnName, String value, long
         return BufferCell.live(cdef, timestamp, ByteBufferUtil.bytes(value));
     }
 
-    private Cell<?> expiring(TableMetadata cfm, String columnName, String value, long timestamp, int localExpirationTime)
+    private Cell<?> expiring(TableMetadata cfm, String columnName, String value, long timestamp, long localExpirationTime)
     {
         return expiring(cfm, columnName, value, timestamp, 1, localExpirationTime);
     }
 
-    private Cell<?> expiring(TableMetadata cfm, String columnName, String value, long timestamp, int ttl, int localExpirationTime)
+    private Cell<?> expiring(TableMetadata cfm, String columnName, String value, long timestamp, int ttl, long localExpirationTime)
     {
         ColumnMetadata cdef = cfm.getColumn(ByteBufferUtil.bytes(columnName));
         return new BufferCell(cdef, timestamp, ttl, localExpirationTime, ByteBufferUtil.bytes(value), null);
     }
 
-    private Cell<?> deleted(TableMetadata cfm, String columnName, int localDeletionTime, long timestamp)
+    private Cell<?> deleted(TableMetadata cfm, String columnName, long localDeletionTime, long timestamp)
     {
         ColumnMetadata cdef = cfm.getColumn(ByteBufferUtil.bytes(columnName));
         return BufferCell.tombstone(cdef, timestamp, localDeletionTime);
     }
+
+    private static void assertThrowsOnInvalidDeletionTime(CheckedFunction runnable)
```

Comment:
Nit: we can slightly simplify this and get better error messages with:
```java
private static void assertThrowsOnInvalidDeletionTime(ThrowableAssert.ThrowingCallable runnable)
{
    Assertions.assertThatThrownBy(runnable)
              .isInstanceOf(IllegalArgumentException.class)
              .hasMessageContaining("is outside the range for an unsigned integer value");
}
```

Reply:
That is what I was looking for and I didn't manage to find originally. Thx.